2.3-tapscript.ipynb

```diff
@@ -366,16 +386,22 @@
     "    * Note: for n < m, empty signature elements (zero) must be provided.\n",
     "    \n",
     "\n",
-    "* `ts(csaolder(k, [key0, key1, ...], hash))`\n",
-    "    * Witness: `[signature], [signature], ..., [delay]`\n",
     "\n",
-    "\n",
-    "* `ts(csahash(k, [key0, key1, ...], hash, time))`\n",
+    "* `ts(csa_hashlock(k, [key0, key1, ...], hash, time))`\n",
     "    * Witness: `[signature], [signature], ..., [32B pre-image]`\n",
+    "    * Hashlock: `hash160(32B-preimage)`\n",
+    "\n",
+    "\n",
+    "\n",
+    "* `ts(csa_delay(k, [key0, key1, ...], hash, time))`\n",
+    "    * Witness: `[signature], [signature], ...`\n",
+    "    * Spendable after delay (with `nSequence > delay`)\n",
     "\n",
     "\n",
-    "* `ts(csahasholder(k, [key0, key1, ...], hash, time))`\n",
-    "    * Witness: `[signature], [signature], ..., [32B pre-image], [delay]`"
+    "* `ts(csa_hashlock_delay(k, [key0, key1, ...], hash, time))`\n",
+    "    * Witness: `[signature], [signature], ..., [32B pre-image], [delay]`\n",
```

Comment:
I don't think the witness includes `[delay]`

Reply:
Fixed in #141.